.ci/Jenkinsfile

```diff
@@ -85,10 +97,23 @@ pipeline {
     }
     post {
         always {
+            script {
+                if(!env.CHANGE_ID) {
+                    snykSecurityScan.generateHtmlElements()
+                }
+            }
             junit testResults: '**/build/test-results/**/*.xml', keepLongStdio: true, allowEmptyResults: true
         }
         cleanup {
             deleteDir()
         }
     }
 }
+
+def isReleaseBranch() {
+    return (env.BRANCH_NAME =~ /^release\/.*$/)
+}
+
+def isMainBranch() {
+    return (env.BRANCH_NAME =~ /^master$/)
+}
```

Comment:
I thought these already existed?

Reply:
The changes here were ported from the main (release/6.0) branch and I was trying to keep them as consistent as possible. As the logic is not being used within the pipeline it can be removed. 
Ref main (release/6.0): https://github.com/corda/corda-gradle-plugins/commit/18d04004d6d6405a790835d6f6ebd3b01e5c20e9

Reply:
Fair enough! What's release/6.0 for?

Reply:
Honestly, I'm not sure